File: app/Global.java
import play.*;
import models.User;

public class Global extends GlobalSettings {

	@Override
	// Add admin account into database if it's not in
	public void onStart(Application app) {
		if (User.find.all().size() == 0) {
			User user = new User(1L, "admin", "fc312767fb4c127a8d2759a5e513b03a33483b21", "admin@aeroclub.git");
			user.save();
		}
	}
}

File: app/Global.scala
import play.api._
import play.api.mvc._
import play.api.mvc.Results._
import views.html._
import models.User

object Global extends GlobalSettings {

	override def onHandlerNotFound(request: RequestHeader): Result = {
		NotFound(views.html.notFound(request))
	}

	override def onStart(app: Application) {
		if(User.find.all.size == 0) {
			val user = new User(1L, "admin", "fc312767fb4c127a8d2759a5e513b03a33483b21", "admin@aeroclub.git")
			user.save()
		}
	}
}